### ISSUE TITLE
Use `$TMP` instead of `/tmp`

### ISSUE THREAD

**Issue by lstor** [twolfson/sexy-bash-prompt#46]
...because `$TMP` should work on Linux, Mac *and* Windows :)

**Comment by twolfson** [comment]
I wasn't really considering Windows but I guess if it makes sense then we can add it. For what it's worth, that one line install is based off of `git-extras` so you should complain to them too =P

https://github.com/tj/git-extras/wiki/Installation#clone--tarball

I will take a look at fixing this by the end of the weekend.

**Comment by lstor** [comment]
Yeah, the one line install is easy to fix manually anyway. I was thinking more of the "My colors don't work as advertised"-script.

With a tiny bit of extra care, I don't see why these things shouldn't work on MsysGit and/or cygwin. And it's pretty neat!
